src/store/reducers/writerProfileReducer.js

```diff
@@ -6,14 +6,17 @@ import {
 
 const initialState = {
   writerProfile: {
-    id: null,
-    first_name: null,
-    last_name: null,
-    website: null,
-    biography: null,
-    education: null,
-    work_history: null,
-    portfolio: null,
+    id: "",
```

Comment:
is this giving you an empty object in your state? If so, you might want to consider and empty object for state? It would depend on the way